frontend/src/components/PregameWelcomeText/PregameWelcomeText.js

```diff
@@ -0,0 +1,224 @@
+          web browser if in person (as if you have real life friends)
+        </li>
+        <li>
+          One player streams this screen to all the others / has this on the
+          shared screen
+        </li>
+        <li>
+          All players go to this site in a web browser, hit the join button, and
+          input the &quot;JOIN CODE&quot; shown in the bottom left of this
+          screen
+        </li>
+        <li>
+          The host selects the card packs to play with, what the winning score
+          should be, and sets a max player limit (if desired)
+        </li>
+        <li>
+          Once ready to start, the host clicks the &quot;START CAROUSING&quot;
+          button on this screen
+        </li>
+      </ol>
+    </>
+  );
+}
+
+function Rules1() {
+  return (
+    <>
+      <h2>Rules:</h2>
+      <p className="snarkyText">
+        (if you want to follow them, I&apos;m not your dad)
+      </p>
+      <ol>
+        <li>
+          Each round 1 player is selected as the Czar, and the other players
+          will be jesters
+        </li>
+        <li>
+          A black card with a sentence will appear on this screen. You might
+          think &quot;how could this card be funny&quot; but that&apos;s just
+          because you have no sense of humor
+        </li>
+        <li>
+          Jesters will each receive a hand of white cards with a unique
+          word/phrase. See above with regards to cards you don&apos;t think are
+          funny
+        </li>
+        <li>The Czar reads the text on the black card out loud</li>
+        <li>
+          Jesters submit a number of their white cards equal to the &quot;PICK
+          X&quot; number on the bottom right of the black card
+        </li>
+      </ol>
+    </>
+  );
+}
+
+function Rules2() {
+  return (
+    <>
+      <h2>Rules 2 - Eccentric Caribou: </h2>
+      <ol start="6">
+        <li>
+          The Czar reads each submission out loud, in the context of the black
+          card... yes every time Janet
+        </li>
+        <li>
+          The Czar picks their favorite submission, the winning jester gets one
+          point
+        </li>
+        <li>
+          A different player becomes Czar and a new round starts. Just think of
+          it like you&apos;re in imperial Russia. TIP: if you start losing claim
+          a communist revolution and give everyone equal points. Except for you
+          of course, as the leader of the revolution you deserve more
+        </li>
+        <div className="howToWin">
+          <h3>The first player to reach the winning score wins!</h3>
+          <p className="snarkyText">(duh)</p>
+        </div>
+      </ol>
+    </>
+  );
+}
+
+export default function PregameWelcomeText() {
+  return (
+    <StyledPregameWelcomeText>
+      <Carousel
+        slides={[<Welcome />, <Setup />, <Rules1 />, <Rules2 />]}
+        interval={2000}
```

Comment:
My feeling on first glance is that these were switching way too fast. Can this increase to like 10 seconds or something? There's a lot of text to read.

Reply:
Yep my plan was to increase this to like a minute or two, but since that would mean reviewers have to wait all that time staring at the screen to verify that it works I was waiting until after at least the initial reviews before I do it.